opentelemetry/proto/trace/v1/trace.proto

```diff
@@ -23,35 +23,37 @@ option java_multiple_files = true;
 option java_package = "io.opentelemetry.proto.trace.v1";
 option java_outer_classname = "TraceProto";
 
-// A span represents a single operation within a trace. Spans can be
+// Span represents a single operation within a trace. Spans can be
 // nested to form a trace tree. Spans may also be linked to other spans
-// from the same or different trace. And form graphs. Often, a trace
+// from the same or different trace and form graphs. Often, a trace
 // contains a root span that describes the end-to-end latency, and one
 // or more subspans for its sub-operations. A trace can also contain
 // multiple root spans, or none at all. Spans do not need to be
 // contiguous - there may be gaps or overlaps between spans in a trace.
 //
-// The next id is 15.
+// The next field id is 17.
 message Span {
-  // A unique identifier for a trace. All spans from the same trace share
-  // the same `trace_id`. The ID is a 16-byte array. An ID with all zeroes
-  // is considered invalid.
+  // trace_id is the unique identifier of a trace. All spans from the same trace share
+  // the same `trace_id`. The ID is a 128 bit (16 byte) value represented using two
+  // fixed64 fields. An ID with value of 0 is considered invalid.
```

Comment:
Not sure I understand. In OpenTelemetry the trace id is defined as a 16 bytes array. So you need to explain how that will be converted to unit128 or directly to the fixed64.

Reply:
I'll add a clarification.

Reply:
I added a clarification, please have a look.